docs/installation.rst

```diff
@@ -0,0 +1,386 @@
+
+.. code-block:: bash
+
+    conda deactivate
+
+Creating a virtual environment with ``venv``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Modern Python versions, regardless of their origin, come with a built in tool to create virtual environments, ``venv``. However, contrary to ``conda`` it does not let you select the version of Python that will be used - it will be the same one you are using to create the environment. Therefore, you still need to make sure your version of Python is compatible with MUSE.
+
+Another caveat is that the virtual environment will be created in a specific folder, so whenever you want to use it in the future, you will need to remember in what folder it was created and activate the environment from there.
+
+You can create a virtual environment with:
+
+.. code-block:: bash
+
+    python -m pip install venv
+    python -m venv venv
+
+And then you activate the environment with:
+
+- Linux:
+
+    .. code-block:: bash
+
+        source venv/bin/activate
+
+- MacOS:
+
+    .. code-block:: zsh
+
+        . venv/bin/activate
+
+- Windows:
+
+    .. code-block:: powershell
+
+        venv\Scripts\Activate.ps1
+
+Later, to recover the system-wide "normal" python, deactivate the environment with:
+
+.. code-block:: bash
+
+    deactivate
+
+Installing MUSE in a virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Regardless of the method used, **once it has been created and activated**, you can install ``MUSE`` within using:
+
+.. code-block:: bash
+
+    python -m pip install muse-os
+
+And then use it by invoking ``muse`` with the relevant input arguments. Keep in mind that, contrary to using ``pipx``, in this case **you will need to manually activate the environment every time you want to use MUSE**.
+
+Instructions for development
+----------------------------
+
+Developers of MUSE will need to have the version control tool ``git`` installed in their system and be familiar with its usage. The `Introduction to Git and GitHub for software development <https://imperialcollegelondon.github.io/introductory_grad_school_git_course/>`_ course created by `Imperial RSE Team <https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/service-offering/research-software-engineering/>`_ can be a good place to start.
+
+Installing MUSE source code in editable mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have ``git`` in your system, clone MUSE repository with:
+
+.. code-block:: bash
+
+    git clone https://github.com/SGIModel/MUSE_OS.git
+
+Then, we will create a virtual environment using `venv`, as explained above:
+
+.. code-block:: bash
+
+    cd MUSE_OS
+    # Create virtual environment - for development, this is typically called ".venv"
+    python -m venv .venv
+    # Activate virtual environment following the instructions for your operative system.
+    # Finally, install MUSE in editable mode:
+    python -m pip install -e .[dev,doc]
+
```

Comment:
I disagree. Anaconda is a monstrosity of 800 Mb that is not required at all to install and run MUSE. It should be here because people might want to use it for other reasons, but setting it as default and forcing people to installing it when that is not required, I don't feel is right. I'm happy to give the two options here, though.

Reply:
We do not need to make Anaconda default, but we should have the same structure as previously done for creating an environment, reporting the options for both, venv and anaconda

Reply:
ok!